marc_to_solr/lib/action_note_builder.rb

```diff
@@ -0,0 +1,91 @@
+class ActionNoteBuilder
+  attr_reader :record
+
+  def initialize(record:)
+    @record = record
+  end
+
+  def self.build(record:)
```

Comment:
I like the API of this class.

Reply:
Thanks! I stole it from the AccessFacetBuilder class, that it looks like @eliotjordan originally wrote!